content/for-developers/tutorials/inner-loop/05-expose-app/expose-app.md

```diff
@@ -1,99 +1,76 @@
-
-a. In your deployment configuration YAML file, define a route:
+Learn how to expose your application to external traffic.
 
-```yaml
-apiVersion: route.openshift.io/v1
-kind: Route
-metadata:
-  name: nordmart-route
-spec:
-  host: nordmart.example.com
-  to:
-    kind: Service
-    name: nordmart-service
-```
+## Key Results
 
-In the above example, a route named `nordmart-route` is defined, pointing to the `nordmart-service` service.
+- Create route
 
-b. Apply the configuration to create the route:
+- Create service based on the application requirements
 
-`oc apply -f route.yaml`
-c. Once the route is created, it will assign a hostname (`nordmart.example.com`) that you can use to access your application externally.
+## Tutorial
 
-Note: In order to use a hostname, you must have a DNS record pointing to the OpenShift cluster's external IP or load balancer.
+Assuming that you already have deployed the application using the previous tutorial.
 
-By following the steps outlined above, you can successfully expose your application deployed on Red Hat OpenShift, using the example application "`nordmart`". Whether using the NodePort, LoadBalancer, or Route approach, you can make your application accessible from outside the OpenShift cluster, allowing users to access and interact with your application over the network.
+OpenShift provides a routing mechanism called "routes" that allows you to expose applications using hostnames and paths. Routes are created using the Route resource and can provide additional features such as SSL termination and path-based routing. To create a route:
 
-## Service Type
+1. In your `tilt/values-local` file, define a route:
 
-In OpenShift, services are used to expose applications internally within the cluster. To expose your application externally, you need to specify the appropriate service type:
+    ```yaml
+    ## Route
+    route:
+      enabled: true
+      port:
+        targetPort: http
+    ```
 
-a. In your deployment configuration YAML file, define a service with the type NodePort or LoadBalancer:
+    a) Route is `enabled: true`, which means the route resource will be created when you tilt up.
 
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-  name: nordmart-service
-spec:
-  type: NodePort
-  ports:
-    - port: 80
-      targetPort: 8080
-  selector:
-    app: nordmart
-```
+    b) `port:` This section defines the port configuration for the Route resource.
 
-In the above example, a service named `nordmart-service` is defined with the NodePort type, exposing port 80 and forwarding traffic to port 8080 of the application.
+    c) `targetPort: http`: This specifies the target port for the Route. In this case, the value is set to `http`, which is a named port defined in the associated Service configuration. It represents the port on which the backend service is listening to handle incoming traffic.
```

Comment:
add a note after this line to with link to application chart for allowing users to see other configurations